### PR TITLE
ISS-197886: Update the mobile SDK callouts

### PR DESCRIPTION
**Pull request by bojan**
## Summary

The mobile SDKs notes were inconsistent, this PR aims to fix them. It also covers the changes proposed by EKLine.

## Connected Issues

- #ISS-197886

## Craftsmanship, Integrity, and Devil’s Advocacy

- [ ] Testing: Negative test cases: null or default values, crash and fault injection tests
- [ ] Testing: Boundary conditions: rolling upgrades, denial-of-service, etc.
- [ ] Testing: Fixing a few existing — flaky or permanently-broken — test cases
- [ ] Observing: Detailed error codes so machines can understand
- [ ] Observing: Adding superior metrics for future debugging
- [ ] Observing: Tracing the hairiest pathways for field serviceability
- [ ] Training: KnowledgeOps update? So AI always works.

### CODE REVIEW

File: fern/docs/pages/sdks/mobile/flutter/quickstart.mdx [hunk 1]
```diff
@@ -50,7 +50,7 @@ DevRev.configure(appID: "abcdefg12345")
 
 A sample app with use cases for the DevRev SDK for Flutter has been provided as a part of our [public repository](https://github.com/devrev/devrev-sdk-flutter). To set up and run the sample app:
```
[the remark this answers is not in the document]
Comment (bojan) [reply fern/docs/pages/sdks/mobile/flutter/quickstart.mdx:51]: The repository is currently private, as the product is getting released as we speak, this comment is irrelevant.
